references.Rmd

```diff
@@ -1,3 +1,4 @@
-`r if (knitr:::is_html_output()) '
 # References {-}
-'`
+
+<!-- Todo: re-add if (knitr:::is_html_output()) bit? -->
```

Comment:
@Robinlovelace maybe try reinstate this? I looked at the changes and I cannot guess what is the source of the issue...

Reply:
Done.